src/pub.c

```diff
@@ -917,9 +917,6 @@ DPS_Status DPS_SendPublication(DPS_Node* node, DPS_Publication* pub, RemoteNode*
              */
             DPS_NetFreeBufs(bufs, 2);
         }
-        if (ret == DPS_OK) {
-            UpdatePubHistory(node, pub);
```

Comment:
Did you test to ensure that this change doesn't have any impact on loop detection and recovery? My concern is that the probe publications sent to detect link loss will be short-circuited due to this change.

Reply:
Update: completed a run of this branch's mesh_stress against hot_mesh.txt with no errors.